showcase/app/templates/page-components/modal.hbs

```diff
@@ -40,10 +40,7 @@
     {{#each @model.COLORS as |color|}}
       <SF.Item @label={{capitalize color}} class="shw-component-modal-sample-item">
         <Hds::Modal open @color={{color}} id="modal-example-{{color}}" as |M|>
-          <M.Header
-            @icon={{concat (if (eq color "warning") "alert-triangle") (if (eq color "critical") "alert-diamond")}}
-            @tagline="Tagline"
-          >
+          <M.Header @icon={{get @model.COLOR_TO_ICON_MAP color}} @tagline="Tagline">
```

Comment:
[question] should this come from the model? this to me belongs to the controller (it's kind of a "logical" function: depending on the `color` give me the corresponding icon)